### DISCUSSION TITLE
Fix scroll bar on `edit`/`preview` code screen

### DISCUSSION THREAD

**Discussion by piradata** [community/community#53086] [Other Feature Feedback, Questions, & Ideas] [answered]
### Fix scroll bar on `edit`/`preview` code screen

[Look at the scroll bar on the `edit` tab](https://cdn.discordapp.com/attachments/910235068436447232/1105860342459945020/image.png)

[Now look at the scroll bar on the `preview` tab](https://cdn.discordapp.com/attachments/910235068436447232/1105860124972695562/image.png)

Should not both be inside the container?

[When I scroll down in the `preview` tab](https://cdn.discordapp.com/attachments/910235068436447232/1105860152462151740/image.png) I can't go back to `edit` unless I scroll all the way to the top before... And that does not happens on the `edit` tab.

**Comment by piradata** [comment]
Also, is this the right place to post bug issues about github itself? Just could not find anywhere for that and the forum seemed the only obvious place that remained...

**Comment by piradata** [comment]
\label bug report

**Reply by piradata** [reply]
anyone? it seems like an actually easy fix

**Comment by piradata** [comment]
still broken...

and it is so easy to fix

![image](https://github.com/community/community/assets/16529503/d157c627-d8de-4b4c-bb64-61f838bdb8c2)

just add this 3 styles on the div that has the markdown rendering. where can i open a pr with this fix @tuves ?

**Reply by Cbodfield** [reply]
@piradata Thanks for the heads up on this and for providing a possible solution! I'll look into getting this fix out soon.

**Comment by Cbodfield** [comment, marked as answer]
The fix for this has been deployed @piradata. Thanks again for reporting!

**Reply by piradata** [reply]
Great! Thanks! XD
